Java3_HW_7/Tests.java

```diff
@@ -0,0 +1,61 @@
+public class Tests {
+    private static final boolean A = true ;
+    private static final boolean B = false ;
+
+    @Test(priority = 2)
+    public static void Test2Lvl(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.